ReactWindows/ReactNative/Views/Slider/ReactSliderManager.cs

```diff
@@ -0,0 +1,151 @@
+        /// <param name="minimum">
+        /// The minimum slider value.
+        /// </param>
+        [ReactProp("minimumValue")]
+        public void SetMinimumValue(Windows.UI.Xaml.Controls.Slider view, double minimum)
+        {
+            view.Minimum = minimum;
+        }
+
+        /// <summary>
+        /// Sets to change slider maximum value.
+        /// </summary>
+        /// <param name="view">a slider view.</param>
+        /// <param name="maximum">
+        /// The maximum slider value.
+        /// </param>
+        [ReactProp("maximumValue")]
+        public void SetMaximumValue(Windows.UI.Xaml.Controls.Slider view, double maximum)
+        {
+            view.Maximum = maximum;
+        }
+
+        /// <summary>
+        /// Sets slider value.
+        /// </summary>
+        /// <param name="view">The slider view.</param>
+        /// <param name="value">
+        /// Slider value.
+        /// </param>
+        [ReactProp(ViewProps.Value)]
+        public void SetValue(Windows.UI.Xaml.Controls.Slider view, double value)
+        {
+            view.Value = value;
+        }
+
+        /// <summary>
+        /// Sets slider step.
+        /// </summary>
+        /// <param name="view">The slider view.</param>
+        /// <param name="step">
+        /// Slider step.
+        /// </param>
+        [ReactProp("step")]
+        public void SetStep(Windows.UI.Xaml.Controls.Slider view, double step)
+        {
+            view.StepFrequency = step;
+        }
+
+        /// <summary>
+        /// This method should return the <see cref="ReactSliderShadowNode"/>
+        /// which will be then used for measuring the position and size of the
+        /// view. 
+        /// </summary>
+        /// <returns>The shadow node instance.</returns>
+        public override ReactSliderShadowNode CreateShadowNodeInstance()
+        {
+            return new ReactSliderShadowNode();
+        }
+
+        public override void UpdateExtraData(Windows.UI.Xaml.Controls.Slider root, object extraData)
+        {
+        }
+
+        /// <summary>
+        /// Returns the view instance for <see cref="Slider"/>.
+        /// </summary>
+        /// <param name="reactContext"></param>
+        /// <returns></returns>
+        protected override Windows.UI.Xaml.Controls.Slider CreateViewInstance(ThemedReactContext reactContext)
+        {
+            return new Windows.UI.Xaml.Controls.Slider();
+        }
+
+        /// <summary>
+        /// Subclasses can override this method to install custom event 
+        /// emitters on the given view.
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        /// <param name="view">The view instance.</param>
+        protected override void AddEventEmitters(ThemedReactContext reactContext, Windows.UI.Xaml.Controls.Slider view)
+        {
+            view.ValueChanged += OnValueChange;
+            view.PointerReleased += OnPointerReleased;
```

Comment:
Nevermind just looked at the Android implementation, looks like they use touch as well.

---
In reply to: [76804165](https://github.com/ReactWindows/react-native-windows/pull/655#discussion_r76804165) [](ancestors = 76804165)